README.md

```diff
@@ -145,6 +145,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Verscend Technologies](http://verscend.com.np/) | Hattisar, Kathmandu | Software research and development center
 - [Vianet Communications Pvt. Ltd.](https://www.vianet.com.np/) | Jawalakhel, Lalitpur | Provider of high speed Internet connectivity and communications solutions for business and individuals
 - [Visualise VR](http://visualisevr.com/) | Pulchowk, Lalitpur |  Help business benefit from mixed reality technologies
+- [Viveka Health](http://www.vivekahealth.com/) | Lalitpur | Outsourcing company (Parent company based in US)
```

Comment:
@SushilShrestha Where exactly in Lalitpur? Something like `Jawlakhel, Lalitpur` would do.

I am trying to make the list very consistent.

Reply:
Also, can you remove `(Parent company based in US)` from the description. Every company seems to be based somewhere and I am trying to remove that bias from the description.

Reply:
parent company has been removed but I have to confirm about the exact location. will update the location after further confirmation.